setup_mysql_replicator_multi.sql

```diff
@@ -29,6 +29,10 @@ CREATE TABLE IF NOT EXISTS `settings` (
   `enable_loose_insert` int(11) DEFAULT '0',
   -- On enabling 'enable_loose_delete: 1', turn on speculative delete but performance penalty on non-contiguous primary key.
   `enable_loose_delete` int(11) DEFAULT '0',
+  -- On enabling 'enable_retry: 1', automatically retries when an error occurs due to MySQL.
+  `enable_retry` int(11) DEFAULT '0',
+  -- Additional interval when retrying. If not set, waits for the time set in the regular interval column.
+  `retry_interval` int(11) NOT NULL DEFAULT 0,
```

Comment:
This SQL is referenced from [Tutorial](https://github.com/y-ken/fluent-plugin-mysql-replicator/blob/master/Tutorial-mysql_replicator_multi.md?plain=1#L22). I would like to recommend to enable by default for starter user.
We believe this change is reasonable as it will not suddenly change the behavior for those who are already using it.
```suggestion
  -- On enabling 'enable_retry: 1', automatically retries when an error occurs due to MySQL.
  `enable_retry` int(11) DEFAULT '1',
  -- Additional interval when retrying. If not set, waits for the time set in the regular interval column.
  `retry_interval` int(11) NOT NULL DEFAULT '30',
```

the reason of retry_interval: 30sec, In environments that rely on error detection on the data pipeline side rather than log file monitoring, it can be assumed that errors will continue to occur for about a day. In order to prevent the log from becoming too large, I thought it would be easier to perform operations if the default setting was to retry at 30-second intervals.